src/modules/ShortcutGuide/ShortcutGuide/shortcut_guide.h

```diff
@@ -78,6 +78,17 @@ class OverlayWindow
         std::wstring value;
     } disabledApps;
 
+    struct ShouldReactToPressedWinKey
+    {
+        static inline PCWSTR name = L"use_legacy_press_win_key_behavior";
+        bool value;
+    } shouldReactToPressedWinKey;
+
+    struct WindowsKeyPressTime
+    {
+        static inline PCWSTR name = L"press_time";
```

Comment:
nit:
```suggestion
        static inline PCWSTR name = L"win_key_press_time";
```

Reply:
The main reason for this is that it will pick up on the previous setting name "press_time", so it'll load this setting for users who had changed it on previous versions.

Reply:
Ah, ok. Fine by me then, let's avoid migration overhead then :)

Reply:
Basically it's the legacy name for the setting.